server/models/message.js

```diff
@@ -144,6 +146,50 @@ module.exports = function(sequelize, DataTypes) {
     });
   }
 
+ function findCurrentRemindersforUser(toUserId) {
+     return Message.findAll({
+      where: { toUserId: toUserId, type: REMINDER_TYPE },
+      include: [{
+        model: models.User,
+        as: "fromUser",
+        required: true
+      }],
+      include: [{
+        model: models.User,
+        as: "toUser",
+        required: true
+      },
```

Comment:
This hash has two entries for "include"!   I imagine that the last one wins!   If you need both, you should combine them.  The include array may contain multiple entries.